app/assets/javascripts/services/table.js

```diff
@@ -0,0 +1,86 @@
+function compareServiceClassification(elementA, elementB) {
+  if (
+    !['A', 'B', 'C', 'D', 'E', 'F', 'N/A'].includes(elementA.dataset.classification)
+    || !['A', 'B', 'C', 'D', 'E', 'F', 'N/A'].includes(elementB.dataset.classification)
+  ) {
+    return 0;
+  }
+  if (elementA.dataset.classification === elementB.dataset.classification) {
+    return 0;
+  }
+
+  // Both do not have the same classification, so if one is good, that one is better:
+  if (elementA.dataset.classification === 'A') {
+    return -1;
+  }
+  if (elementB.dataset.classification === 'A') {
+    return 1;
+  }
+  // Both do not have the same classification, and neither is good, so if one is neutral, that one is better:
+  if (elementA.dataset.classification === 'B') {
+    return -1;
+  }
+  if (elementB.dataset.classification === 'B') {
+    return 1;
+  }
+  // Both do not have the same classification, and neither is good or neutral, so if one is bad, the other is a blocker:
+  if (elementA.dataset.classification === 'C') {
+    return -1;
+  }
+  if (elementB.dataset.classification === 'C') {
+    return 1;
+  }
+  if (elementA.dataset.classification === 'D') {
+    return -1;
+  }
+  if (elementB.dataset.classification === 'D') {
+    return 1;
+  }
+  if (elementA.dataset.classification === 'E') {
+    return -1;
+  }
+  if (elementB.dataset.classification === 'E') {
+    return 1;
+  }
+  if (elementA.dataset.classification === 'F') {
+    return -1;
+  }
+  if (elementB.dataset.classification === 'F') {
+    return 1;
+  }
+  if (elementA.dataset.classification === 'N/A') {
+    return -1;
+  }
+  if (elementB.dataset.classification === 'N/A') {
+    return 1;
+  }
```

Comment:
I strongly suspect there is a shorter way to write this... :-)

Reply:
Do make sure that, if you submit a PR, the shorter way is also at least as readable as the current way - code is read more often than it is written! :)